pipeline/tasks/protoc_tasks.py

```diff
@@ -206,20 +206,22 @@ class ProtoDescGenTask(task_base.TaskBase):
     """Generates proto descriptor set"""
     default_provides = 'descriptor_set'
 
-    def execute(self, src_proto_path, import_proto_path, output_dir,
-                api_name, toolkit_path):
-        print 'Compiling descriptors {0}'.format(
-            _find_protos(src_proto_path))
+    def execute(self, src_proto_path, desc_proto_path, import_proto_path,
```

Comment:
`desc_proto_path` could be an optional parameter (move at the end with `desc_proto_path = None` notation, and then initialize with an empty list when `None`). In that case that is not a `GAPIC_REQUIRED` parameter, but it would simplify the artman config a bit, because, anyways, `desc_proto_path` would be empty in many APIs.